src/test/java/com/sap/sgs/phosphor/fosstars/model/score/oss/UnpatchedVulnerabilitiesScoreVerification.java

```diff
@@ -0,0 +1,15 @@
+package com.sap.sgs.phosphor.fosstars.model.score.oss;
+
+import com.sap.sgs.phosphor.fosstars.model.qa.VerificationFailedException;
+import java.io.IOException;
+import org.junit.Test;
+
+public class UnpatchedVulnerabilitiesScoreVerification {
+
+  @Test
+  public void verify() throws IOException, VerificationFailedException {
+    UnpatchedVulnerabilitiesScore.Verification.createFor(
```

Comment:
As discussed, I think we can remove the parameter for createFor() ?

Reply:
I forgot to mention that the `UnpatchedVulnerabilitiesScore` is not a singleton so that there may be multiple instances of this class. Right now, we use the `Verification` class to verify only `OssScores.UNPATCHED_VULNERABILITIES` but in the future we may want to verify other instances. That's why we may need this parameter in the `createFor()` method.

Reply:
Ok, makes sense then .